elastic/kibana
# [Infra] Add new metrics tab and redesign of hosts overview metrics section

closes [#175446](https://github.com/elastic/kibana/issues/175446)
## Summary

Adds new Metrics tab and redesigns the metrics section in the Overview tab



https://github.com/elastic/kibana/assets/2767137/766305ce-23c6-4001-ab20-2230decc02a1




### Extra changes
 - Replaced usages of asset name in favor of asset id to filter asset data across the Asset Details components - which will help when it starts supporting more asset types
 - Renamed some translation keys to use `assetDetails` instead of `nodeDetails`

### How to test
- Start a local Kibana instance
- Navigate to Infrastructure > Hosts
- Open the flyout for a host
  - Overview tab 
    - Check if the metrics section matches the design and AC
  - Metrics tab
    - Check if it matches the design the design and AC
- Check the above in the full page view.
